gt/lua_lpm.c

```diff
@@ -382,6 +382,81 @@ l_lpm6_lookup(lua_State *l)
 	return 1;
 }
 
+/*
+ * Takes an array of uint8_t (IPv6 address) and masks it using the depth.
+ * It leaves untouched one bit per unit in the depth variable
+ * and set the rest to 0.
+ */
+static inline void
+ip6_mask_addr(uint8_t *ip, uint8_t depth)
```

Comment:
This function isn't so tiny to deserve the `inline` qualifier.

Moreover, don't we already have a function that computes masked addresses in Gatekeeper?